cli/internal/scm/scm_test.go

```diff
@@ -15,8 +15,8 @@ func TestGetCurrentBranchMain(t *testing.T) {
 	testDir := getTestDir(t, "myrepo")
 
 	// Setup git
-	gitCommand(t, testDir, []string{"config", "--global", "user.email", "turbo@vercel.com"})
-	gitCommand(t, testDir, []string{"config", "--global", "user.name", "Turbobot"})
+	gitCommand(t, testDir, []string{"config", "--local", "user.email", "turbo@vercel.com"})
```

Comment:
Oh, they're new. 🤦‍♂️

Reply:
I started with `--global`, because on Windows and Linux CI matrix, `git commit` spouted a bit message asking me to set the user and recommended this. But then when I ran the tests locally, it (obviously) also set _my_ global config, ruining all future commits of my real work. 

So I opted to make it local instead. Interestingly:

- the `--local` flag did not work in CI (on the same windows/linux CI matrix boxes), so I had to drop the flag altogether
- the macOS CI job doesn't require setting the user/email. Not sure why that is